### PR TITLE
Add "elsewhere" option to fetch, pull and push menu

### PR DESCRIPTION
**Pull request by rzikm**
Implements #139.

### CODE REVIEW

File: src/ops/push.rs [hunk 1]
```diff
@@ -25,3 +25,22 @@ impl OpTrait for Push {
         }))
     }
 }
+
+#[derive(Display)]
+#[display(fmt = "Push elsewhere")]
```
[the remark this answers is not in the document]
Comment (rzikm) [reply src/ops/push.rs:30]: I see it with `e` binding

image

I have `evil` keybindings though.

[the remark this answers is not in the document]
Comment (altsem) [reply src/ops/push.rs:30]: Ah, this is when i stepped to your latest commit without creating a matching branch!

Detached HEAD:
image
Checked out branch:
image

**Comment by rzikm** [comment]
We should be ready for another round of review

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,13 @@
 
 All notable changes to this project will be documented in this file.
 
+## [unreleased]
+
+### 🚀 Features
+
+- Add "elsewhere" option to fetch, pull and push menu
+- Syntax highlighting for Elixir
+
```
Comment (altsem) [inline CHANGELOG.md:11]: Ah, I should've noted in my suggestion: this is generated when I make a release. So the commit [Add changelog entry](https://github.com/altsem/gitu/pull/160/commits/e45cd82595e89e2d38019c214dc3fdc865633acd) wouldn't be needed.

**Review by altsem — APPROVED** [review]
Looks good!

File: CHANGELOG.md (hunk 2 above)
Comment (rzikm) [reply CHANGELOG.md]: oh, I already force-pushed to remove the commit

Comment (altsem) [reply CHANGELOG.md]: :+1: Ah you're fast! My merge failed because of it. Let's include all the commits then :)

**Review by altsem — APPROVED** [review]

**Pull request merged**